web/frontend/src/components/base.vue

```diff
@@ -45,16 +53,14 @@ const angle = ref(0);
 const videoStreamStates = new Map<string, boolean>();
 const selectCameras = ref('');
 
+const pressed = new Set<Keys>();
```

Comment:
We now add all pressed keys to a stack of unique items, and them digest those items from oldest to newest input. This allows us to better handle multiple keys pressed at the same time.